networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -562,17 +645,26 @@ def purge_infranetworks_on_metagroup(cls, metagroup: conf.Hostgroup, member_hgs:
             raise ValueError("'member_hgs.binding_host[0]' and metagroup.members are not matching")
 
         minimum_common_infra_nets = set(member_hgs[0].infra_networks)
+        minimum_common_extra_vlans = set(member_hgs[0].extra_vlans if member_hgs[0].extra_vlans else [])
```

Comment:
could be `minimum_common_extra_vlans = set(member_hgs[0].extra_vlans or []`, if you want it shorter.

Reply:
I did not know. Thanks.